pkg/helm/helpers.go

```diff
@@ -23,13 +23,11 @@ func (t *testKubernetesFactory) GetClient(configPath string) (kubernetes.Interfa
 // NewTestMixin initializes a helm mixin, with the output buffered, and an in-memory file system.
 func NewTestMixin(t *testing.T) *TestMixin {
 	c := context.NewTestContext(t)
-	m := &TestMixin{
-		Mixin: &Mixin{
-			Context:       c.Context,
-			ClientFactory: &testKubernetesFactory{},
-		},
+	m := New()
```

Comment:
This ensures that values on the mixin struct aren't accidentally not initailized. So instead of using the struct directly, I'm using its New function.